### DISCUSSION TITLE
High CPU load in development mode in v13

### DISCUSSION THREAD

**Discussion by pecca** [vercel/next.js#47159] [Help]
### Summary

After upgrading Next.js v12.3.1 -> v13.2.3 we have noticed high CPU load problem in the development mode. After a short period of random use, the CPU load increases to above 400% and it doesn't recover without a restart. Any idea how this could be fixed.

There are no problems in production mode.

**Comment by ytianzaishui** [comment]
This is a big question, I also meet.
Open office getStaticProps like web https://static-tweet.vercel.app/ or any website dev by next ex: https://redux.js.org/api/store
CUP up!!!
![image](https://user-images.githubusercontent.com/18025841/236980617-06ffb0a9-b4ad-4d4a-b197-f644299f6c81.png)
